infra/chronos/manager.py

```diff
@@ -21,31 +21,47 @@
 import json
 import subprocess
 
-import integrity_validator_check_replay
-import integrity_validator_run_tests
+try:
+  from chronos import integrity_validator_check_replay
+  from chronos import integrity_validator_run_tests
+except ImportError:
+  import integrity_validator_check_replay
+  import integrity_validator_run_tests
 
 logger = logging.getLogger(__name__)
 
 
+def _get_oss_fuzz_root():
+  """Gets the root directory of the OSS-Fuzz repository."""
+  return os.path.abspath(
+      os.path.join(os.path.dirname(__file__), os.pardir, os.pardir))
+
+
 def _get_project_cached_named(project, sanitizer='address'):
   """Gets the name of the cached project image."""
-  return f'us-central1-docker.pkg.dev/oss-fuzz/oss-fuzz-gen/{project}-ofg-cached-{sanitizer}'
+  base_name = 'us-central1-docker.pkg.dev/oss-fuzz/oss-fuzz-gen'
+  return f'{base_name}/{project}-ofg-cached-{sanitizer}'
 
 
 def _get_project_cached_named_local(project, sanitizer='address'):
   return f'{project}-origin-{sanitizer}'
 
 
-def build_project_image(project):
-  """Build OSS-Fuzz base image for a project."""
+def build_project_image(project) -> None:
```

Comment:
this seems very similar to what's in helper.py already.

Without looking too carefully, it also feels like a lot of this file has similarities with what's in helper.py 

Is there a way we can have a common place for utilities like building project images?

Maybe we need to break things out into a `project_utils.py` that both `helper.py` and this depends on?